### ISSUE TITLE
mp.nametags.set color wrong type (1.1)

### ISSUE THREAD

**Issue by witschel** [CocaColaBear/types-ragemp-c#66]
Change color: [0, 0, 0, 0]; to color: RGBA;

```
interface NametagsMp {
	enabled: boolean;

	set(style: {
		font: number;
		outline: boolean;
		offset: number;
		veh_offset: number;
		color: RGBA;
		size: number;

		hbar?: {
			size: [number, number];
			color: [0, 0, 0, 0];
			bg_color: RGBA;
		}
	}): void;
}
```

**Comment by notlancer** [comment]
fixed in #67

**Comment by ynhhoJ** [comment]
Closed, thx @Lancer8883
